README.md

```diff
@@ -36,11 +39,13 @@ In this version, the following DSC Resources are already built and ready for use
 - All base DSC resources found in PowerShell 5 ([WMF 5.0][wmf-5.0]).
 - All DSC resources found in the [Microsoft PowerShell DSC Resource Kit][DSCResources]
 
-## Windows System Prerequisites
+## [Windows System Prerequisites][wmf5-blog-requirements]
```

Comment:
Remove this please. The prerequisites section is for the module itself, not for WMF5. Thanks!